torch/_dynamo/polyfill.py

```diff
@@ -0,0 +1,10 @@
+"""
+Python polyfills for common builtins.
+"""
+
+
+def all(iterator):
+    for elem in iterator:
+        if not elem:
+            return False
+    return True
```

Comment:
Couldn't this be `return builtins.all(iterator)`?

Reply:
NO! The whole point is that builtins.all is implemented in C, so Dynamo cannot trace through it, while this is implemented in pure Python, so we can just Dynamo trace through it.

Reply:
Ah, I see now the "inline_user_function_return". So this is just desugaring the function. Is it an issue that we are using here an `if` statement?

Reply:
No, that will just cause guards which is fine.

I suppose it could be a problem if some things are unbacked symints; in that case, in principle a non-branching implementation of all that doesn't short circuit could help, but this wouldn't really be preserving Python semantics so we'd have to be very careful about when we attempted this.